crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -872,7 +873,15 @@ pub fn pad_config(node: &Node) -> PadConfig {
                     Some(Data::Float64(constant_value)) => Some(*constant_value as f32),
                      _ => panic!("Pad: only float values are currently supported for constant value, submit an issue on github"),
                 })
-                .unwrap_or(0.0)
+                .unwrap_or(0.0);
+
+        if node.attrs.contains_key("value") {
+            constant_value = node.attrs.get("value").and_then(|value| match value {
```

Comment:
Quick clippy suggestion:

```
error: using `Option.and_then(|x| Some(y))`, which is more succinctly expressed as `map(|x| y)`
   --> crates/burn-import/src/onnx/op_configuration.rs:879:30
    |
879 |               constant_value = node.attrs.get("value").and_then(|value| match value {
    |  ______________________________^
880 | |                 AttributeValue::Float32(value) => Some(*value),
881 | |                 _ => panic!("Pad: only float32 values are currently supported for constant value as attribute, submit an issue on github"),
882 | |             }).unwrap_or(0.0);
    | |______________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#bind_instead_of_map
    = note: `-D clippy::bind-instead-of-map` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::bind_instead_of_map)]`
help: try
    |
879 ~             constant_value = node.attrs.get("value").map(|value| match value {
880 ~                 AttributeValue::Float32(value) => *value,
    |
```

So you can just apply the same match statement in `.map(...)` but you don't need to wrap the result in `Some(*value)` because map will only be applied when value is present.